alacritty/src/display/content.rs

```diff
@@ -420,14 +422,19 @@ struct Hint<'a> {
     labels: &'a Vec<Vec<char>>,
 }
 
+enum HintType {
+    Label(char, bool),
+    AfterLabel,
+}
```

Comment:
First of all I think it would probably make sense to break this out into three different variants, that way we don't have to match on the boolean after matching on the enum.

I also think that `HintPosition` might be a better name.